ControlRoomApplication/ControlRoomApplication/GUI/DiagnosticsForm.Designer.cs

```diff
@@ -298,6 +298,7 @@ private void InitializeComponent()
             this.AmbHumidValidation = new System.Windows.Forms.ToolTip(this.components);
             this.AccelOffsetsValidation = new System.Windows.Forms.ToolTip(this.components);
             this.SensorNetworkPeriodValidation = new System.Windows.Forms.ToolTip(this.components);
+            this.PushEmailNotif_checkBox = new System.Windows.Forms.CheckBox();
```

Comment:
I'm just gonna hope that this file is all good because 1299 changes is a lot, LGTM